### PR TITLE
Features/eng 392

### PR DESCRIPTION
**Pull request by eolszewski**
View demo here - https://dashboard.transmute.network/demo
Be sure to connect to ropsten with metamask.

### CODE REVIEW

File: packages/transmute-framework/src/event-store-factory/index.js [hunk 1]
```diff
@@ -49,9 +49,13 @@ module.exports = class EventStoreFactory {
 
     this.version = pack.version;
 
-    this.web3 = new Web3(
-      new Web3.providers.HttpProvider(web3Config.providerUrl)
-    );
+    if (window.web3) {
```
Comment (OR13) [inline packages/transmute-framework/src/event-store-factory/index.js]: does this break node.js compatibility? We used to have some smoke tests for verifying this kind of thing.

File: packages/transmute-framework/src/event-store/index.js [hunk 2]
```diff
@@ -163,7 +167,12 @@ module.exports = class EventStore {
       }
     );
 
-    const receipt = web3.eth.getTransactionReceipt(tx);
+    const receipt = web3.eth.getTransactionReceipt(tx, function (error, result) {
+      if (!error)
+        console.log(result)
```
Comment (OR13) [inline packages/transmute-framework/src/event-store/index.js]: console.info ?

Comment (eolszewski) [reply packages/transmute-framework/src/event-store/index.js]: Good call.

File: packages/transmute-framework/src/event-store-factory/index.js (hunk 1 above)
Comment (eolszewski) [reply packages/transmute-framework/src/event-store-factory/index.js]: Adding this

**Pull request merged**